src/status_im2/contexts/syncing/standard_authentication/style.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im2.contexts.syncing.standard-authentication.style)
+
+(def main-container
+  {:width  "100%"})
+
+(def enter-password-container
+  {:margin-horizontal        20
+   :border-top-left-radius   12
+   :border-top-right-radius  12})
```

Comment:
There's extra spacing in map columns, probably `make lint-fix` should fix these kind of formatting issues